buildqtc.sh

```diff
@@ -306,6 +317,7 @@ build_unix_qtc() {
             ${OPT_VARIANT_PRETTY:+"QTCREATOR_DISPLAY_VERSION='$OPT_VARIANT_PRETTY'"} \
             "DEFINES+=IDE_COPY_SETTINGS_FROM_VARIANT=." \
             "DEFINES+=IDE_SETTINGSVARIANT=$OPT_VARIANT" \
+            ${OPT_COPY_FROM_VARIANT:+"DEFINES+=IDE_COPY_SETTINGS_FROM_VARIANT=$OPT_COPY_FROM_VARIANT"} \
```

Comment:
Now IDE_COPY_SETTINGS_FROM_VARIANT gets added twice: First with value . on line 318 and then with the wanted value on line 320.

Reply:
I never realized we already did so :) It was needed before https://github.com/sailfishos/sailfish-qtcreator/commit/dedbb40f57e936c9d8188c1c96c4156a1059bc12 it seems. Fixed.